sdk/core/Azure.Core.Experimental/tests/public/Azure.Core.Experimental.Tests.Public.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <!--<IncludeGeneratorSharedCode>true</IncludeGeneratorSharedCode>-->
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Moq" />
+    <PackageReference Include="NUnit" />
+    <PackageReference Include="NUnit3TestAdapter" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="$(AzureCoreTestFramework)" />
+    <ProjectReference Include="..\..\src\Azure.Core.Experimental.csproj" />
+  </ItemGroup>
+  
+  <ItemGroup>
+    <Compile Remove="perf/**/*.*" />
+  </ItemGroup>
```

Comment:
I don't think this is needed since we are in a peer directory of `perf/`

Reply:
Good catch - thanks!